shared/storage/minio.py

```diff
@@ -143,50 +162,64 @@ def create_root_storage(self, bucket_name="archive", region="us-east-1"):
     # Writes a file to storage will gzip if not compressed already
     def write_file(
         self,
-        bucket_name,
-        path,
-        data,
-        reduced_redundancy=False,
+        bucket_name: str,
+        path: str,
+        data: BinaryIO,
+        reduced_redundancy: bool = False,
         *,
-        is_already_gzipped: bool = False,
+        is_already_gzipped: bool = False,  # deprecated
+        is_compressed: bool = False,
+        compression_type: str = "zstd",
     ):
+        if is_already_gzipped:
+            log.warning(
```

Comment:
this would spam warnings for *all* of the calls setting this flag.
As it should be fairly trivial to migrate code over to the new flags, this should probably not log any warnings or sentry messages, but just perform the fallback.

Ideally the migration path would be 4 PRs:
- this one
- migrate `is_already_gzipped` users within `worker`
- migrate `is_already_gzipped` users within `api`
- remove the flag and backwards compatibility code